tests/exporters/tflite/test_exporters_tflite_cli.py

```diff
@@ -65,8 +66,13 @@ def _get_models_to_test(export_models_dict: Dict):
                         exporter_config_kwargs=DEFAULT_DUMMY_SHAPES,
                     )
 
+                    if task == "question-answering":
+                        default_shapes = dict(DEFAULT_DUMMY_SHAPES)
+                        default_shapes["sequence_length"] = 384
+                    else:
+                        default_shapes = DEFAULT_DUMMY_SHAPES
```

Comment:
why?

Reply:
I am trying to avoid that, currently preprocessing fails for some reason (`stride < max_len`) and related errors.